src/services/anchor-service.ts

```diff
@@ -326,6 +326,10 @@ export default class AnchorService {
         throw new Error(`No valid ceramic document found with docId ${docId.toString()}`)
       }
 
+      if (doc.tip.toString() != request.cid) {
```

Comment:
```suggestion
      if (doc.tip.toString() != request.cid.toString()) {
```
To be consistent with the logging below.

Reply:
request.cid is already a string, not a CID object